Create FAQ.md (#7954)
iptv-org/iptv: FAQ.md

- [I've created my channel and I want to add it to this playlist. What should I do?](#Ive-created-my-channel-and-i-want-to-add-it-to-this-playlist-what-should-i-do)

Technical:
- [Why am I asked to provide an adaptive playlist like "master.m3u8", "playlist.m3u8" or "index.m3u8"?](#Why-am-i-asked-to-provide-an-adaptive-playlist-like-masterm3u8-playlistm3u8-or-indexm3u8)
- [Why don't we accept XStreamCode in streams?](#Why-dont-we-accept-xstreamcode-in-streams)
- [Is it possible to add streams from Youtube, Dailymotion or Twitch?](#Is-it-possible-to-add-streams-from-youtube-dailymotion-or-twitch)
- [Is it possible to add audio-only streams, like FM radio?](#Is-it-possible-to-add-audio-only-streams-like-fm-radio)
- [Why are there some ids with call sign (WATB-TV.us) and others with alphanumeric id (Mychannel.us)?](#Why-are-there-some-ids-with-call-sign-watb-tvus-and-others-with-alphanumeric-id-mychannelus)
- [There are source for verify call sign and coverage area for some TV stations?](#There-are-source-for-verify-call-sign-and-coverage-area-for-some-tv-stations)
- [Why some streams have empty id?](#Why-some-streams-have-empty-id)

> - [What is a "daily update" and why is benefit for this playlist?](#What-is-a-daily-update-and-why-is-benefit-for-this-playlist)

Reviewer: ```suggestion
- [What is a "daily update" and what are the benefits for this playlist?](#What-is-a-daily-update-and-what-are-the-benefits-for-this-playlist)
```

### Why are there some call sign (KJLA-DT1.us) as ids and alphanumeric names as ids (Mychannel.us)?
It's to differentiate the origin of the broadcast from the content. Call signs usually come from physical stations in the country of origin and their programming is subject to change. If the channel is not from a television station, the brand name will be used.

An example of a call sign used as id is KJLA-DT1.us: [KJLA](https://en.wikipedia.org/wiki/KJLA) is a digital television station, DT is a suffix, 1 is a subchannel number and ".us" is the country code (United States). This station broadcasts the Visión Latina channel, whose id is VisionLatina.us.

### There are source for verify call sign and coverage area for some TV stations?
There are websites to consult the location and characteristics of the station. For example, [FCCData](https://fccdata.org/) for US, Canada, Mexico UK, Austrailia and Japan.

### Why some streams have empty id?
Because the stream does not have enough information available. If you know, let us know its official name, the language it broadcasts and the country where it originates.

> ### What is a "daily update" and why is benefit for this playlist?

Reviewer: ```suggestion
### What is a "daily update" and why are the benefits for this playlist?
```

> A "daily update" is maintenance to check the status of the playlist operation, as well as to hide streams from the playlist if they have access problems.

Reviewer: ```suggestion
A "daily update" is a pull request made for maintenance purposes. It checks the status of the playlist operation, and reclassifies the streams internally. This will be useful on playlist recreation, as it's gonna hide streams from the playlist if they have access problems.
```

NSFW content are not suitable for the main public. To avoid issues with these kind of programmings, we've created a separate NSFW playlist available for everyone. If you want to see this type of channel, please use the NSFW playlist.

Based on [#1799](https://github.com/iptv-org/iptv/pull/1799) and [#3031](https://github.com/iptv-org/iptv/pull/3031).

### Do I have to pay to use this list?
No. Also, the maintenance of this service is self-financed.

### Are you planning to include a VOD of TV shows broadcasted on this playlist?
No. This is not our purpose.

Based on [#175](https://github.com/iptv-org/iptv/issues/175), [#3290](https://github.com/iptv-org/iptv/discussions/3290), [#7852](https://github.com/iptv-org/iptv/discussions/7852).

## Channels
### Does the channel need to be publicly accessible to be included in the playlists?
Yes, we need the links to be publicly accessible like FTA packages (free-to-air), online services or official websites. We recommend tagging [Geo-blocked] if it is only accessible in one or few countries.

Based on [#480](https://github.com/iptv-org/iptv/issues/480).

### Can I add a FTA channel that broadcasts PPV events or pay programming?

> Maybe. Consider if the country where it is broadcasted allows fair use. Owners can ban the broadcast of PPV events, and even issue a DMCA takedown. Check the [blocklist](https://github.com/iptv-org/database/blob/master/data/blocklist.csv) for more info about specific events, and own [counternotice](https://github.com/github/dmca/blob/3ce1c9402c6b7e46dcbbea6ff866d6a44e75c72a/2020/10/2020-10-08-dfl-counternotice.md) indicating that a DCMA takedown is invalid if only one FTA channel infringes copyright.

Reviewer: ```suggestion
Maybe. Consider if the country where it is broadcasted allows fair use. Owners can ban the broadcast of PPV events, and even issue a DMCA takedown. Check the [blocklist](https://github.com/iptv-org/database/blob/master/data/blocklist.csv) for more info about specific events, and this [counternotice](https://github.com/github/dmca/blob/3ce1c9402c6b7e46dcbbea6ff866d6a44e75c72a/2020/10/2020-10-08-dfl-counternotice.md) indicating that a DMCA takedown is invalid if only one FTA channel infringes copyright. Please remember that by sharing a link to a FTA channel, you are **indexing** the link, not **restreaming** their content. This subtility makes most of the takedowns false and invalid. Some valid takedowns do occur when a Pull Request is made to notify us that we should not index their channels..
```

- [Why don't we accept Xtream-Codes servers inside our playlists?](#Why-dont-we-accept-xstreamcodes-servers-inside-our-playlists)
- [Is it possible to add streams from Youtube, Dailymotion or Twitch?](#Is-it-possible-to-add-streams-from-youtube-dailymotion-or-twitch)
- [Is it possible to add audio-only streams, like FM radio?](#Is-it-possible-to-add-audio-only-streams-like-fm-radio)
- [Why are there some ids with call sign (WATB-TV.us) and others with alphanumeric id (Mychannel.us)?](#Why-are-there-some-ids-with-call-sign-watb-tvus-and-others-with-alphanumeric-id-mychannelus)
- [Why attributes of channels of this playlist like "tvg-country", "tvg-language", "tvg-logo" are missing?](#Why-attributes-of-channels-of-this-playlist-like-tvg-country-tvg-language-tvg-logo-are-missing)
- [There are source for verify call sign and coverage area for some TV stations?](#There-are-source-for-verify-call-sign-and-coverage-area-for-some-tv-stations)
- [Why some streams have an empty id?](#Why-some-streams-have-an-empty-id)
- [What is a "daily update" and what are the benefits for this playlist?](#What-is-a-daily-update-and-what-are-the-benefits-for-this-playlist)

## Basic

> ### What is IPTV?
> In non-technical words, IPTV (Internet Protocol television) stations have web addresses that *are obtained from official sources or authorized services for easier access to the end user who has any media player that supports streaming*.
> 
> See [Internet Protocol television](https://en.wikipedia.org/wiki/Internet_Protocol_television) from Wikipedia article. This article is important, because newcomers do not understand how playlists work.
> 
> ### Do I need a good Internet connection to watch an available stream?
> A lot of channels are ready for main public because of adaptive resolution. If you have a slow connection, play again at another time, because you may experience short stuttering and/or signal loss.
> 
> Some TV stations have a maximum of simultaneous users. So if this stream does not display correctly (becuase of bandwidth limit), you will have to play at another time.
> 
> ### I have a cable service and I want those channels to be on this IPTV. Is that possible?
> No. This playlist does not replace traditional cable, because this playlist can't include exclusive channels that may not be available on free-to-air. We recommend contacting your cable provider to offer legal alternatives to this service or visit the [iptv-org/awesome-iptv](https://github.com/iptv-org/awesome-iptv) repository for find other providers.
> 
> ### Does the playlist have a channel guide?
> Yes. However, not all channels have their EPG available. See [EPG](https://github.com/iptv-org/epg) for more info. Please note that the guide is not perfect and some channels (like international feeds) may interrupt programming without you knowing.
> 
> ### Is it possible to view channels directly from the main website?
> No, to avoid possible abuse of the service.
> 

Reviewer 2: ```suggestion
```

> ### Why don't we show NSFW channels in most playlists?

Author: So, do you plan to add this option, and why?

> ### Why don't we show NSFW channels in most playlists?

Reviewer 2: In fact, initially the playlists included all channels. The changes you're referring to here were made only after users started asking to remove adult channels from all playlists.

> ### Why don't we show NSFW channels in most playlists?

Reviewer: Then I think we should reference the first complaints about the X channels.

NSFW content are not suitable for the main public. To avoid issues with these kind of programmings, we've created a separate NSFW playlist available for everyone. If you want to see this type of channel, please use the NSFW playlist.

Based on [#1799](https://github.com/iptv-org/iptv/pull/1799) and [#3031](https://github.com/iptv-org/iptv/pull/3031).

> ### Do I have to pay to use this list?
> No. Also, the maintenance of this service is self-financed.
> 

Reviewer 2: ```suggestion
```

Author: So, why is OpenCollective there?

### Are you planning to include a VOD of TV shows broadcasted on this playlist?
No. This is not our purpose.

Based on [#175](https://github.com/iptv-org/iptv/issues/175), [#3290](https://github.com/iptv-org/iptv/discussions/3290), [#7852](https://github.com/iptv-org/iptv/discussions/7852).

## Channels
### Does the channel need to be publicly accessible to be included in the playlists?
Yes, we need the links to be publicly accessible like FTA packages (free-to-air), online services or official websites. We recommend tagging [Geo-blocked] if it is only accessible in one or few countries.

Based on [#480](https://github.com/iptv-org/iptv/issues/480).

> ### Can I add a FTA channel that broadcasts PPV events or pay programming?
> Maybe. Consider if the country where it is broadcasted allows fair use. Owners can ban the broadcast of PPV events, and even issue a DMCA takedown. Check the [blocklist](https://github.com/iptv-org/database/blob/master/data/blocklist.csv) for more info about specific events, and this [counternotice](https://github.com/github/dmca/blob/3ce1c9402c6b7e46dcbbea6ff866d6a44e75c72a/2020/10/2020-10-08-dfl-counternotice.md) indicating that a DMCA takedown is invalid if only one FTA channel infringes copyright.
> 
> Please remember that by sharing a link to a FTA channel (or more), you are **indexing** the link, not **restreaming** its content. This subtility makes most of the DMCA takedowns false and invalid under law.
> A good example of this would be [Perfect 10 v. Google](https://www.eff.org/fr/cases/perfect-10-v-google). 
> Some [valid takedowns](https://github.com/iptv-org/iptv/issues/6486) do occur when a Pull Request is made to notify us that we should not index their channels.
> 
> Based on [#6350](https://github.com/iptv-org/iptv/issues/6350).
> 

Reviewer 2: ```suggestion
```

> ### I've created my channel and I want to add it to this playlist. What should I do?
> We are a community initiative and you don't need to pay anything to stay on this playlist. Feel free to create a channel with few resources:
> * Camera with a tripod (recommended for your own studio).
> * A PC with Internet, video editor and recording/broadcasting app under RTMP protocol (e.g. OBS).
> * And a live streaming service with multibitrate (if possible).
> 
> After you have your m3u8 link, add your channel information onto the [Database](https://github.com/iptv-org/database) repository, and submit an [issue](https://github.com/iptv-org/iptv/issues/new) to propose it to us.
> 
> Note: There are cases of broadcasting a copy of third-party content (released previously in public) with the purpose of promoting, educating or commenting. This is known as fair use (also called "fair dealing" or "copyright exception") and should be important when broadcasting without problems on your TV channel.
> 

Reviewer 2: ```suggestion
```

Author: I think it is a common sense question. About no pay for add, that's what we are always doing.

> ### Is it possible to limit the broadcasting of my channel to my country only?

Reviewer 2: I can't find the origin of the question. At the link, a man asks to remove from playlists channels blocked in his country.

Author: One example is about a TV news channel that use to third party material for commentary. Do you understand what I'm saying?

Reviewer 2: I'm sorry, I thought you wanted to collect the actual frequently asked questions from the whole repository together with the answers and just put them in one file, that's the only reason I asked for the source.

Since it's just your own q&a you don't need to add any links of course.

Author: This question is more for those who wish to create their own television station. As the creators tend to be more from Latin America (I guess, for online streaming), I doubt that they will ask questions in English for Github (just a channel from Argentina as evidence). So I found it useful.

Reviewer: > This question is more for those who wish to create their own television station. As the creators tend to be more from Latin America (I guess, for online streaming), I doubt that they will ask questions in English for Github (just a channel from Argentina as evidence). So I found it useful.

Don't hesitate, if you're onto translating, to make one Spanish-translated FAQ. I could make a French one

Author: This PR must be finished first, and then the translation must be done. We need to complete the community consensus to continue.

If you do not want the channel to be broadcast outside your country (for legal reasons), we recommend using the [Geo-blocked] tag for your stream.

Based on [#6534](https://github.com/iptv-org/iptv/issues/6534).

> ### Is there any way to add my channel other than via this git repository?
> No, there is no such option.
> 

Reviewer 2: ```suggestion
```

Based on [#1916](https://github.com/iptv-org/iptv/issues/1916).

### Why don't we accept Xtream-Codes servers inside our playlists?
Xtream-Codes streams are, most of the times, pirated streams created by some people that can get satellite and/or PPV streams and that propose it to everyone for a fairer price. These types of servers are illegal and also highly unstable, since it depends either from the user who bought it or the server that hosts it.

In short, they are poor quality streams created by unauthorized people.

If you're unsure about if your link is from an Xtream-Codes server, you may:
* Look at the structure of the URL. Most of them have this form : http(s)://*hostname*:25461/*username*/*password*/*channelID* (port is often 25461)
* Remove the rest of the URL after the password mention and add either "panel_api.php?" or "player_api.php?" after the port. Replace the slashes between the username and password with "username=" and "password=". 

If the link matches or answers after changing the URL, you're with an Xtream-Codes server.

Based on [#5401](https://github.com/iptv-org/iptv/pull/5401).

> ### Is it possible to add streams from Youtube, Dailymotion or Twitch?
> Yes. Due to technical limitations, it is necessary to add a [Streamlink](https://streamlink.github.io/) server link to view the content.
> 
> ### Is it possible to add audio-only streams, like FM radio?
> No. Exceptions are the visual radios, in which a video and audio are shown at the same time. A WIP repository made by one member of iptv-org, [LaneSh4d0w](https://github.com/LaneSh4d0w) aims to collect radio streams, on the [IPRD](https://github.com/LaneSh4d0w/IPRD) repository.
> 
> ### Why are there some call sign (KJLA-DT1.us) as ids and alphanumeric names as ids (Mychannel.us)?
> It's to differentiate the origin of the broadcast from the content. Call signs usually come from physical stations in the country of origin and their programming is subject to change. If the channel is not from a television station, the brand name will be used.
> 
> An example of a call sign used as id is `KJLA-DT1.us`: [KJLA](https://en.wikipedia.org/wiki/KJLA) is a digital television station, DT is a suffix, 1 is a subchannel number and ".us" is the country code (United States). This station broadcasts the Visión Latina channel, whose id is `VisionLatina.us`.
> 
> ### There are source for verify call sign and coverage area for some TV stations?
> There are websites to consult the location and characteristics of the station. For example, [FCCData](https://fccdata.org/) for US, Canada, Mexico, UK, Australia and Japan.
> 

Reviewer 2: ```suggestion
```

Author: I added several page about these questions. The last question is understandable because it is for those who need to find a source for television stations.

### Why attributes of channels of this playlist like "tvg-country", "tvg-language", "tvg-logo" are missing?
The reason is to reduce the workload when adding streams in the list. Since "tvg-id" links to the [Database](https://github.com/iptv-org/database) repository, each channel has unique attributes like image, name (in English and local speak), country (or countries) broadcasted, category and language.

You can add multiple streams with a single id (in the "tvg-id" parameter), instead of adding information from scratch. For example:

```

> #EXTINF:-1 tvg-id="CCTV3.cn" status="online",CCTV-3综艺 (1080p)

Reviewer 2: ```suggestion
#EXTINF:-1 tvg-id="CCTV3.cn",CCTV-3综艺 (1080p)
```

The `status` can also be omitted: https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#stream-description-scheme

```
Based on [#2086](https://github.com/iptv-org/iptv/issues/2086) (countries) and [#6516](https://github.com/iptv-org/iptv/issues/6516) (use of Database).

### Why some streams have an empty id?
Because the stream does not have enough information available. If you appear to know additional data on these, let us know its official name, the language it broadcasts and the country where it originates.

This also applies to the undefined.m3u file. The streams in this file are from channels whose country of origin is unknown.

Based on [#2440](https://github.com/iptv-org/iptv/issues/2440).

### What is a "daily update" and why are the benefits for this playlist?
A "daily update" is a pull request made for maintenance purposes. It checks the status of the playlist operation, and reclassifies the streams internally. This will be useful on playlist recreation, as it's gonna hide streams from the playlist if they have access problems.

> 
> Based on [#3057](https://github.com/iptv-org/iptv/issues/3057).

Reviewer 2: ```suggestion
```

The link refers to a change in the `scripts/clean.js`, it is not part of the "daily update".

Author: Done.

In short, they are poor quality streams created by unauthorized people.

If you're unsure about if your link is from an Xtream-Codes server, you may:
* Look at the structure of the URL. Most of them have this form : http(s)://*hostname*:25461/*username*/*password*/*channelID* (port is often 25461)
* Remove the rest of the URL after the password mention and add either "panel_api.php?" or "player_api.php?" after the port. Replace the slashes between the username and password with "username=" and "password=". 

If the link matches or answers after changing the URL, you're with an Xtream-Codes server.

Based on [#5236](https://github.com/iptv-org/iptv/pull/5236), [#5401](https://github.com/iptv-org/iptv/pull/5401) and [#7576](https://github.com/iptv-org/iptv/discussions/7576) (about username and password of Xtream-Codes server).

### Is it possible to add streams from Youtube, Dailymotion or Twitch?

> Yes. Due to technical limitations, it is necessary to add a [Streamlink](https://streamlink.github.io/) server link to view the content. This does not work with all streams. This does not work with all streams, so it will give an "error" status if this stream is not freely accessible.

Reviewer: ```suggestion
Yes. Due to technical limitations, it is necessary to add a [Streamlink](https://streamlink.github.io/) server link to view the content. This does not work with all streams, so it will give an "error" status if this stream is not freely accessible.
```
remove duplicate sentence

Author: OK. My apologies. I just had to say that geoblocked streams don't work with Streamlink.

Reviewer: > OK. My apologies. I just had to say that geoblocked streams don't work with Streamlink.

It's just because you've said it in your next sentence that I have removed this one :)

# Frequently Asked Questions

Basic:
- [I have a cable service and I want those channels to be on this IPTV. Is that possible?](#I-have-a-cable-service-and-i-want-those-channels-to-be-on-this-iptv-is-that-possible)
- [Is possible to view channels directly from the main website?](#Is-possible-to-view-channels-directly-from-the-main-website)
- [Why don't we show NSFW channels in most playlists?](#Why-dont-we-show-nsfw-channels-in-most-playlists)
- [Do I have to pay to use this list?](#Do-i-have-to-pay-to-use-this-list)
- [Are you planning to include a VOD of TV shows broadcasted on this playlist?](#Are-you-planning-to-include-a-vod-of-tv-shows-broadcasted-on-this-playlist)

Technical:
- [Does the channel need to be FTA to be included in the playlists?](#Does-the-channel-need-to-be-FTA-to-be-included-in-the-playlists)
- [Can I add a FTA channel that broadcasts PPV events or pay programming?](#Can-I-add-a-FTA-channel-that-broadcasts-PPV-events-or-pay-programming)

> - [Why is recommended the "playlist.m3u" or "index.m3u" at the end of the url?](#Why-is-recommended-the-playlistm3u-or-indexm3u-at-the-end-of-the-url)

Reviewer 3: ```suggestion
- [Why is recommended the "master.m3u", "playlist.m3u" or "index.m3u" at the end of the url?](#Why-is-recommended-the-playlistm3u-or-indexm3u-at-the-end-of-the-url)
```

Author: Ok. Thanks.

### Is possible to view channels directly from the main website?
No, to avoid possible abuse of the service.

### Why don't we show NSFW channels in most playlists?
Because to avoid content that may be inappropriate for work or display to the entire audience. We suggest using the corresponding playlist.

### Do I have to pay to use this list?
No. Also, the maintenance of this service is self-financed.

### Are you planning to include a VOD of TV shows broadcasted on this playlist?
No. This is not our purpose.

## Technical
### Does the channel need to be publicly accessible to be included in the playlists?
Yes, we only need the links to be publicly accessible like FTA packages (free-to-air), online services or official websites. We recommend tagging [Geo-blocked] if it is only accessible in one or few countries.

### Can I add a FTA channel that broadcasts PPV events or pay programming?
Maybe. Consider if the country where it is broadcasted allows fair use. Owners can ban the broadcast of PPV events, and even DCMA takedowns. Check the blocklist from the repository database for more info about specific events.

> ### Why is recommended the "playlist.m3u" or "index.m3u" at the end of the url?

Reviewer 3: ```suggestion
### Why is recommended the "master.m3u", "playlist.m3u" or "index.m3u" at the end of the url?
```

### Are you planning to include a VOD of TV shows broadcasted on this playlist?
No. This is not our purpose.

## Technical
### Does the channel need to be publicly accessible to be included in the playlists?
Yes, we only need the links to be publicly accessible like FTA packages (free-to-air), online services or official websites. We recommend tagging [Geo-blocked] if it is only accessible in one or few countries.

### Can I add a FTA channel that broadcasts PPV events or pay programming?
Maybe. Consider if the country where it is broadcasted allows fair use. Owners can ban the broadcast of PPV events, and even DCMA takedowns. Check the blocklist from the repository database for more info about specific events.

### Why is recommended the "master.m3u8", "playlist.m3u8" or "index.m3u8" at the end of the url?
These files are useful to see perfectly the channel based in your network or screen (adaptive resolution). Also the streams are permanent, without token, that avoids edit temporally their urls.

### Why don't we accept XStreamCode in streams?
XStreamCode contains code that could be malicious to users.

### Is possible to add streams from Youtube, Dailymotion or Twitch?

> Yes. Due to technical limitations is necessary add a Streamlink server link to view the content. 

Reviewer 2: ```suggestion
Yes. Due to technical limitations is necessary add a [Streamlink](https://streamlink.github.io/) server link to view the content.
```

# Frequently Asked Questions

Basic:
- [I have a cable service and I want those channels to be on this IPTV. Is that possible?](#I-have-a-cable-service-and-i-want-those-channels-to-be-on-this-iptv-is-that-possible)

> - [Is possible to view channels directly from the main website?](#Is-possible-to-view-channels-directly-from-the-main-website)

Reviewer: ```suggestion
- [Is it possible to view channels directly from the main website?](#Is-it-possible-to-view-channels-directly-from-the-main-website)
```

Author: Good sentence.

- [Does the playlist have a channel guide?](#Does-the-playlist-have-a-channel-guide)
- [Are you planning to include a VOD of TV shows broadcasted on this playlist?](#Are-you-planning-to-include-a-vod-of-tv-shows-broadcasted-on-this-playlist)
- [Why don't we show NSFW channels in most playlists?](#Why-dont-we-show-nsfw-channels-in-most-playlists)
- [Do I have to pay to use this list?](#Do-i-have-to-pay-to-use-this-list)

Channels:
- [Does the channel need to be FTA to be included in the playlists?](#Does-the-channel-need-to-be-FTA-to-be-included-in-the-playlists)
- [Can I add a FTA channel that broadcasts PPV events or pay programming?](#Can-I-add-a-FTA-channel-that-broadcasts-PPV-events-or-pay-programming)

> - [I wish created my channel and add it to this playlist. What should I do?](#I-wish-created-my-channel-and-add-it-to-this-playlist-what-should-i-do)

Reviewer: ```suggestion
- [I've created my channel and I want to add it to this playlist. What should I do?](#Ive-created-my-channel-and-i-want-to-add-it-to-this-playlist-what-should-i-do)
```

Technical:
- [Why is recommended the "master.m3u8", "playlist.m3u8" or "index.m3u8" at the end of the url?](#Why-is-recommended-the-masterm3u8-playlistm3u8-or-indexm3u8-at-the-end-of-the-url)
- [Why don't we accept XStreamCode in streams?](#Why-dont-we-accept-xstreamcode-in-streams)

> - [Is possible to add streams from Youtube, Dailymotion or Twitch?](#Is-possible-to-add-streams-from-youtube-dailymotion-or-twitch)

Reviewer: ```suggestion
- [Is it possible to add streams from Youtube, Dailymotion or Twitch?](#Is-it-possible-to-add-streams-from-youtube-dailymotion-or-twitch)
```

- [There are source for verify call sign and coverage area for some TV stations?](#There-are-source-for-verify-call-sign-and-coverage-area-for-some-tv-stations)
- [Why some streams have empty id?](#Why-some-streams-have-empty-id)
- [What is a "daily update" and why is benefit for this playlist?](#What-is-a-daily-update-and-why-is-benefit-for-this-playlist)

## Basic
### I have a cable service and I want those channels to be on this IPTV. Is that possible?
No. This playlist does not replace traditional cable, because this playlist can't include exclusive channels that may not be available on free-to-air. We recommend contacting your cable provider to offer legal alternatives to this service or visit the [iptv-org/awesome-iptv](https://github.com/iptv-org/awesome-iptv) repository for find other providers.

### Does the playlist have a channel guide?
Yes. However, not all channels have their EPG available. See [EPG](https://github.com/iptv-org/epg) for more info. Please note that the guide is not perfect and some channels (like international feeds) may interrupt programming without you knowing.

> ### Is possible to view channels directly from the main website?

Reviewer: ```suggestion
### Is it possible to view channels directly from the main website?
```

No, to avoid possible abuse of the service.

### Why don't we show NSFW channels in most playlists?

> Because to avoid content that may be inappropriate for work or display to the entire audience. We suggest using the corresponding playlist.

Reviewer: ```suggestion
NSFW content are not suitable for the main public. To avoid issues with these kind of programmings, we've created a separate NSFW playlist available for everyone. If you want to see this type of channel, please use the NSFW playlist.
```

### Do I have to pay to use this list?
No. Also, the maintenance of this service is self-financed.

### Are you planning to include a VOD of TV shows broadcasted on this playlist?
No. This is not our purpose.

## Channels
### Does the channel need to be publicly accessible to be included in the playlists?
Yes, we only need the links to be publicly accessible like FTA packages (free-to-air), online services or official websites. We recommend tagging [Geo-blocked] if it is only accessible in one or few countries.

### Can I add a FTA channel that broadcasts PPV events or pay programming?
Maybe. Consider if the country where it is broadcasted allows fair use. Owners can ban the broadcast of PPV events, and even DCMA takedowns. Check the blocklist from the repository database for more info about specific events.

> ### I wish created my channel and add it to this playlist. What should I do?

Reviewer: ```suggestion
### I've created my channel and I want to add it to this playlist. What should I do?
```

We are a community initiative and you don't need to pay anything to stay on this playlist. Feel free to create a channel with few resources:
* Camera with a tripod (recommended for your own studio).
* A PC with Internet, video editor and recording/broadcasting app under RTMP protocol (e.g. OBS).
* And a live streaming service with multibitrate (if possible).

> After you have your m3u8 link, add your channel information from [Database](https://github.com/iptv-org/database).

Reviewer: ```suggestion
After you have your m3u8 link, add your channel information onto the [Database](https://github.com/iptv-org/database) repository, and submit an [issue](https://github.com/iptv-org/iptv/issues/new) to propose it to us.
```

## Technical
### Why is recommended the "master.m3u8", "playlist.m3u8" or "index.m3u8" at the end of the url?
These files are useful to see perfectly the channel based in your network or screen (adaptive resolution). Also the streams are permanent, without token, that avoids edit temporally their urls.

### Why don't we accept XStreamCode in streams?

> XStreamCode contains code that could be malicious to users.

Reviewer: ```suggestion
Xtream-Codes streams are, most of the times, pirated streams created by some people that can get satellite and/or PPV streams and that propose it to everyone for a fairer price. These types of servers are illegal and also highly unstable, since it depends either from the user who bought it and the server that hosts it.
```

No. This playlist does not replace traditional cable, because this playlist can't include exclusive channels that may not be available on free-to-air. We recommend contacting your cable provider to offer legal alternatives to this service or visit the [iptv-org/awesome-iptv](https://github.com/iptv-org/awesome-iptv) repository for find other providers.

### Does the playlist have a channel guide?
Yes. However, not all channels have their EPG available. See [EPG](https://github.com/iptv-org/epg) for more info. Please note that the guide is not perfect and some channels (like international feeds) may interrupt programming without you knowing.

### Is it possible to view channels directly from the main website?
No, to avoid possible abuse of the service.

### Why don't we show NSFW channels in most playlists?
NSFW content are not suitable for the main public. To avoid issues with these kind of programmings, we've created a separate NSFW playlist available for everyone. If you want to see this type of channel, please use the NSFW playlist.

### Do I have to pay to use this list?
No. Also, the maintenance of this service is self-financed.

> ### Are you planning to include a VOD of TV shows broadcasted on this playlist?
> No. This is not our purpose.

Reviewer 2: ```suggestion
### Are you planning to include a VOD of TV shows broadcasted on this playlist?
> No. This is not our purpose.
Based on #175, #3290, #7852
```

### Why don't we show NSFW channels in most playlists?
NSFW content are not suitable for the main public. To avoid issues with these kind of programmings, we've created a separate NSFW playlist available for everyone. If you want to see this type of channel, please use the NSFW playlist.

> If NSFW channels is displayed by mistake, we suggest that you report via issue or edit their entries from [Database](https://github.com/iptv-org/database) repository.

Reviewer: ```suggestion
If NSFW channels are displayed by mistake, we suggest that you report via an issue or edit their entries from the [Database](https://github.com/iptv-org/database) repository.
```

Author: Fixed. I added this sentence to explain some error in the playlists, but I forgot to add "in the general playlist" for more clear.
